examples/pipelines/ccfraud_vertical_fedonce/submit.py

```diff
@@ -0,0 +1,412 @@
+"""Federated Learning Cross-Silo Vertical basic pipeline for ccfraud.
```

Comment:
the pipeline will fail if the number of silos is above 3, can we either 1) issue an exception if the number of silos is not 3 during this submit.py or 2) make the number of embeddings in traininsilo flexible with a maximum number (ex: 5) and optional inputs?